openreview/conference/builder.py

```diff
@@ -436,7 +446,9 @@ def open_bids(self):
         return self.__create_bid_stage()
 
     def close_bids(self):
-        return self.__expire_invitation(self.get_bid_id())
+        self.__expire_invitation(self.get_bid_id(self.get_reviewers_id()))
```

Comment:
^ this is a good example of what I meant in my previous comment. here, `__expire_invitation` has an invitation ID as a parameter, which makes sense to me